packages/axios-extension/src/abap/adt-catalog/handlers/transport.ts

```diff
@@ -36,7 +37,7 @@ const enum AdtTransportStatus {
  * @param log Service provider logger
  * @returns available transport numbers
  */
-function getTransportChecksResponse(doc: Document, xml: string, log: Logger): string[] {
+function getTransportChecksResponse(doc: Document, xml: string, log: Logger): TransportRequest[] {
```

Comment:
What does this function do? Does it return valid transport requests? Can the name be changed to what it does please?